packages/create-sitecore-jss/src/templates/nextjs/next.config.js

```diff
@@ -29,6 +29,16 @@ const nextConfig = {
   // Enable React Strict Mode
   reactStrictMode: true,
 
+  // enable image provider domains and device sizes
+  images: {
+    domains: getHostName([jssConfig.sitecoreApiHost]),
```

Comment:
Seems odd that `getHostName` accepts an array. I'd suggest keeping that as a single string parameter, and change this to:
```suggestion
    domains: [getHostName(jssConfig.sitecoreApiHost)],
```

Reply:
So in case of multiple domain names you want to call the function multiple times?